### PR TITLE
Added function to open recycle bin

### PR DESCRIPTION
**Pull request by tdoerrich**

### CODE REVIEW

**Comment by lances101** [comment]
@SweBeat I am sorry, but both me and @happlebao are busy with other projects and issues. Once development resumes we will review your PR. Thank you for your contribution.

File: Plugins/Wox.Plugin.Sys/Languages/zh-cn.xaml [hunk 1]
```diff
@@ -14,6 +14,8 @@
     <system:String x:Key="wox_plugin_sys_setting">设置</system:String>
     <system:String x:Key="wox_plugin_sys_sleep">休眠这台电脑</system:String>
     <system:String x:Key="wox_plugin_sys_emptyrecyclebin">Empty recycle bin</system:String>
```
Comment (tinymins) [inline Plugins/Wox.Plugin.Sys/Languages/zh-cn.xaml]: 清空回收站

File: Plugins/Wox.Plugin.Sys/Languages/zh-cn.xaml [hunk 2]
```diff
@@ -14,6 +14,8 @@
...
+    <system:String x:Key="wox_plugin_sys_openrecyclebin">Missing translation for: Open recycle bin</system:String>
```
Comment (tinymins) [inline Plugins/Wox.Plugin.Sys/Languages/zh-cn.xaml]: 打开回收站

**Review by tinymins — CHANGES_REQUESTED** [review]
Add Chinese Translation

**Comment by jhdxr** [comment]
closed since author has abonded this PR (source repo not exists)

**Pull request closed without merging**